File: python/projects/project 04/models/doctor.py
from models.employee import Employee

class Medico:
    def __init__(self, funcionario: Employee, crm: str) -> None:
        self.funcionario = funcionario
        self.crm = crm

    def __str__(self) -> str:
        return (
                f"\nMédico"
                f"\n{self.funcionario}"
                f"\nCRM: {self.crm}"
                )

File: java/object-oriented programming/encapsulation/encapsulation13/doctor.java
package com.mycompany.mavenproject3;

public class Medico extends Funcionario{
    private String crm;

    public Medico(String crm, String nome, String cpf, String rg, double salario) {
        super(nome, cpf, rg, salario);
        this.crm = crm;
    }

    public String getCrm() {
        return crm;
    }

    public void setCrm(String crm) {
        this.crm = crm;
    }

    @Override
    public String toString() {
        return "\nMedico" + 
               super.toString() +
               "\nCrm: " + crm;
    }
}
